lib/http-context.js

```diff
@@ -456,6 +462,11 @@ function toXML(input) {
       }
     });
   }
+  if (xmlOptions.wrapperElement === false) {
+    //allowing XML output to have no root element
+    var match = xml.match(/<response>([\s\S]*?)<\/response>$/);
+    xml = match[1].replace(/^[\s]{0,2}/gim, '').trim();
+  }
```

Comment:
removing this because we probably shouldn't support `non-xml` outputs while sending the `XML header`